wp-parsely.php

```diff
@@ -1392,14 +1379,18 @@ public function parsely_add_amp_actions() {
 	public function parsely_add_amp_analytics( $analytics ) {
 		$options = $this->get_options();
 
+		if ( ! empty( $options['disable_amp'] ) && true === $options['disable_amp'] ) {
+			return;
+		}
+
 		if ( empty( $options['apikey'] ) ) {
 			return $analytics;
 		}
 
 		$analytics['parsely'] = array(
-			'type'        => 'parsely',
-			'attributes'  => array(),
-			'config_data' => array(
+			'type'       => 'parsely',
+			'attributes' => array(),
+			'config'     => array(
```

Comment:
See here for discussion of how `config` may be preferred to `config_data` https://github.com/Automattic/amp-wp/issues/1498